ztp/du-profile-kube-compare/required/cluster-tuning/monitoring-configuration/ReduceMonitoringFootprint.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
```

Comment:
In a subsequent version we will need to handle this content. It will be merged with other content from ACM (typically by users) and we want to make sure that these values persist.